client/web/src/nav/StatusMessagesNavItem.tsx

```diff
@@ -279,6 +266,19 @@ export const StatusMessagesNavItem: React.FunctionComponent<React.PropsWithChild
                             />
                         )
                     }
+                    if (status.__typename === 'NoRepositoriesDetected') {
+                        return (
+                            <StatusMessagesNavItemEntry
+                                key="no-repositories"
+                                title="No repositories detected"
```

Comment:
```suggestion
                                title="No repositories"
```

I think that sounds more... human? 😄 We also say "no files in folder" and not "no files in folder detected", I thik.